microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,147 +5,223 @@ author: MSEdgeTeam
+* Do not attempt to cheat users, the system or the ecosystem.  There is no place in Microsoft Edge Add-ons for any kind of fraud; be it ratings and review manipulation, credit card fraud or other fraudulent activity.
 
-The _Microsoft Edge Add-ons store_ is also called the _Microsoft Edge Add-ons website_.  The _Microsoft Edge Add-ons store developer policies_ are also called the _Microsoft Edge Add-ons website developer policies_.
+Adhering to the Microsoft Edge Add-ons store developer policies helps you make choices that enhance the appeal and audience of your extension.
 
 
 <!-- ====================================================================== -->
-## Principles
+## Building quality products
 
-A few principles to get you started:
+Microsoft is committed to delivering quality products to its customers.  Therefore, extensions which don't follow the quality guidelines aren't allowed, such as:
 
-*   You should offer unique and distinct value within your extensions for Microsoft Edge.  Provide a compelling reason to download your extensions from the Microsoft Edge Add-ons store (Microsoft Edge Add-ons).
+* Content deemed not family-friendly.
 
-*   You must not mislead our joint users about what your extension does, who is offering it, and so on.
+* Bots: Any extension that automatically generates spam-like or unsolicited messages on the user's behalf or that which performs automated actions to manipulate a platform's features, such as artificially inflating engagement metrics.
 
-*   You must not attempt to cheat users, the system or the ecosystem.  There is no place in our Microsoft Edge Add-ons for any kind of fraud; be it ratings and review manipulation, credit card fraud or other fraudulent activity.
+* Cryptocurrency: Crypto mining and any other illegitimate activities that involve crypto wallets and currency.
 
-Adhering to the Microsoft Edge Add-ons store developer policies should help you make choices that enhance the appeal and audience of your extension.
+* Non-production builds: For example, an extension that is still in an experimental stage, prone to crashes or instability and any extension that is designed for internal testing purposes only and not intended for public release.
 
-Your extensions are crucial to the experience of hundreds of millions of users.  We look forward to experiencing what you create and are thrilled to help deliver your extensions to the world.
+* Prohibited products: An extension that promotes or facilitates the sale of illegal drugs, firearms, or counterfeit goods would be considered a prohibited product and also any extension that violates intellectual property rights, such as offering pirated software or copyrighted content without authorization, would be restricted.
+
+* Gambling content.
 
 
 <!-- ====================================================================== -->
-## 1. Product Policies
+## 1. Product policies
+
+<!-- todo: add content between headings -->
```

Comment:
2.  Style guides say need content between headings. but these headings are a numbered outline, so does that apply? what could we put here between headings?

Reply:
it's an outline, keeping as-is